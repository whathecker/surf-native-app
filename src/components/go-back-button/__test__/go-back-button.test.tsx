import * as React from "react";
import renderer from "react-test-renderer";
import GoBackButton from "../go-back-button";

describe("Test GoBackButton component", () => {
    it("should renders GoBackButton component correctly", () => {
        const tree = renderer.create(<GoBackButton />).toJSON();

        expect(tree).toMatchSnapshot();
    });
});