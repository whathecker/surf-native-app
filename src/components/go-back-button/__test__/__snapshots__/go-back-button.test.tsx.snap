// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test GoBackButton component should renders GoBackButton component correctly 1`] = `
<View
  accessible={true}
  focusable={true}
  onClick={[Function]}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "marginLeft": 20,
      "marginTop": 55,
      "opacity": 1,
    }
  }
>
  <Image
    source={1}
    style={
      Object {
        "height": 32,
        "width": 32,
      }
    }
  />
</View>
`;
